backend/ttrs/urls.py

```diff
@@ -4,7 +4,9 @@
 from . import views
 
 urlpatterns = [
+    path('students/signin/', views.StudentSignIn.as_view()),
```

Comment:
What is the difference of 'students/<int:pk>/' and 'students/signin/'?

Reply:
`students/signin/` has no need to send the id of the student.
But `students/\<int:pk\>/` should send the id of the student by url.

However, `StudentDetail` cannot be accessed unless the requester is the student.
So, I decided to merge two APIViews.